lib/pages/dashboard/widgets/dashboard_club_teams.dart

```diff
@@ -89,7 +82,8 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
                             team: state.teams[index],
                             distance: _currentTeamPage - index,
                             onTap: () => Router.push(context: context, builder: (_) => TeamDetailPage(team: state.teams[index]))
-                                .then((_) => widget.onTeamFavoriteChange(state.teams[index])),
+                                .then((_) => widget.onTeamsChange(null)
```

Comment:
It seems that the parameters of `onTeamChange` is always `null` now (which is fine as it was not really used before). Instead of using `null` everywhere, it's better to remove this parameter from the callback definition.

Reply:
It's not always `null`, when teams are fetched from the API for the first time, it returns them.